packages/docs/modules/ROOT/pages/network-js.adoc

```diff
@@ -0,0 +1,302 @@
+= Building an OpenZeppelin Network.js powered dapp from scratch
+In this tutorial, we will integrate *OpenZeppelin Network.js* from the OpenZeppelin SDK, to build a dapp that can be used in a browser with injected web3 e.g. using MetaMask. 
+
+The *OpenZeppelin Network.js* is an easy to use and reliable library that provides one line access to the web3 API.  It can be used in both React and in (vanilla) JavaScript.  It also supports the Gas Station Network.
+
+We will use the `create-react-app` package to bootstrap a React application and use *OpenZeppelin Network.js* to easily set up a web3 context. 
+
+== Setting up the environment
+We begin by creating a new project.
+
+[source,console]
+----
+mkdir web3-dapp && cd web3-dapp
+npm init -y
+----
+
+Then we install *OpenZeppelin Network.js*
+[source,console]
+----
+npm install @openzeppelin/network
+----
+
+== Creating the dapp
+We will create our dapp using the `create-react-app` package, which bootstraps a simple client-side application using React.
+
+[source,console]
+----
+npx create-react-app client
+----
+
+Our dapp will display the current Ethereum network that we are connected to and which web3 provider is being used.
+
+In the code below, first we import `useWeb3` from the React implementation of *OpenZeppelin Network.js* (`@openzeppelin/network/react`).
+We then get a web3Context using `useWeb3` which will attempt to retrieve an injected web3 provider (e.g. MetaMask), otherwise it will fall back to a network connection, which in this case is a connection to mainnet via Infura.
```

Comment:
```suggestion
We then get a web3Context using `useWeb3`. This hook will attempt to retrieve an injected web3 provider (e.g. MetaMask), otherwise it will fall back to a network connection, which in this case is a connection to mainnet via Infura.
```